apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1471,5 +1471,77 @@ describe("getSchedule", () => {
         { dateString: plus2DateString }
       );
     });
+    test("include booked events before minimumBookingNotice with seatsMinimumBookingNotice", async () => {
```

Comment:
This is a rather simple test case.
Todo: add a test case that tests a complex scenario